wayland.c

```diff
@@ -400,6 +411,27 @@ bool init_wayland(struct mako_state *state) {
 		}
 	}
 
+	// Set up the cursor. It needs a wl_surface with the cursor loaded into it.
+	// If one of these fail, mako will work fine without the cursor being able to change.
+	state->cursor_theme = wl_cursor_theme_load(NULL, 24, state->shm);
```

Comment:
Use `XCURSOR_SIZE` from the env to allow users to override the cursor size. See https://github.com/emersion/slurp/blob/e77e1c432bcbd101fd084448bab2df173154a0dc/main.c#L889